src/components/Todo/index.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import EditionForm from './EditionForm';
+import DueDate from './DueDate';
+import Dropdown from '../library/Dropdown';
+
+export default class Todo extends React.Component {
+
+  render() {
+    const { todo } = this.props;
+
+    if (this.props.editing) {
+      return (
+        <span className="todo-container">
+          <EditionForm
+            todo={todo}
+            onUpdate={this.props.onUpdate}
+            onCancel={this.props.onCancelEdit}
+          />
+        </span>
+      )
+    }
+
+    return (
+      <span className="todo-container">
+
+        <span
+          style={{
+            textDecoration: todo.active ? "none" : "line-through"
+          }}
+          onClick={() => this.props.onUpdate(todo.id, { active: !todo.active})}
+        >
+          {todo.name}
+        </span>
+
+
+
+        <DueDate dueDate={todo.dueDate}/>
+
+        <Dropdown/>
```

Comment:
Ok, so that's the idea, except we need to pass props to it right ? A dropdown does not necessary contain an Edit and a Delete button. Plus, we need to tell the component what to do when the elements of the dropdown are clicked